### DISCUSSION TITLE
Craft 4

### DISCUSSION THREAD

**Discussion by Mosnar** [trendyminds/craft-palette#17] [Q&A] [answered]
Any plans for a Craft 4 version? If you're open to it, I could probably do a PR in the next week or so.

**Comment by aaronbushnell** [comment, marked as answer]
Hey Mosnar, absolutely! In fact, we're looking to get this out the door this week!

**Reply by aaronbushnell** [reply]
Craft 4.0 support is now in place with 2.0! Reach out if you encounter any bugs!

**Reply by Mosnar** [reply]
Awesome, thanks so much!
